src/devices/ExplorerHat/samples/ExplorerHat.Sample.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <ApplicationIcon />
```

Comment:
Still seeing `<ApplicationIcon />` and `<StartupObject />`. Please match the content of this and other projects

Reply:
I'm sorry, I didn't understand. It's done.